### ISSUE TITLE
Prepare maven release 

### ISSUE THREAD

**Issue by dhuebner** [eclipse/Xpect#267]
deploy a release build to maven central.
Workflow:
- create branch named 'release_x.y.z'
- update versions
- push the branch
- Jenkins will trigger a build and publish to maven staging
- review and publish staging repo from sonatype web UI

**Comment by dhuebner** [comment]
Forst maven warmup release: `https://repo1.maven.org/maven2/org/eclipse/xpect/`

**Comment by cdietrich** [comment]
nice. will there be a release update site at eclipse too?

**Comment by dhuebner** [comment]
@cdietrich Yes, as soon as wir prepared a p2 release. This was a maven warm-up

**Comment by dhuebner** [comment]
A readme will be available here:

https://github.com/eclipse/Xpect/tree/master/org.eclipse.xpect.releng/README.md

**Comment by cdietrich** [comment]
the current maven artifacts dont have dependencies, thus the user has to make sure the relevant ones are present themselves
